### ISSUE TITLE
breaking: discord.js types are broken after newest changes

### ISSUE THREAD

**Issue by T0TProduction** [discordjs/discord.js#3690]
**Please describe the problem you are having in as much detail as possible:**
Our CI threw this right after #3530 was merged into master
This failed us both on new staging as well as old, long-tested production builds that installed the new version on build
![image](https://user-images.githubusercontent.com/18548570/72269527-0511da00-3624-11ea-9c64-ed54b80a5e6b.png)

**Include a reproducible code sample here, if possible:**
install discord.js
use typescript
try to compile

**Further details:**
- discord.js version: current as of the commit mentioned
- Node.js version: 10.x
- Priority this issue should have – please be realistic and elaborate if possible: high

- [x] I have also tested the issue on latest master, commit hash: 8014ddcd1c651749fc64e8dfc0b0863422d57203

**Comment by kyranet** [comment]
Fixed in #3689

**Comment by T0TProduction** [comment]
You should definitely add TS compilation to the CI tests, then this type of error can never happen from merging into master.

**Comment by iCrawl** [comment]
Should also probably not rely on master for production 👀 
But yes, generally this will be done.

**Comment by T0TProduction** [comment]
The production part was tested and mentioned to make sure that no changes on our codebase could have been the cause 😁 

Nice 👍
